scripts/create.ts

```diff
@@ -40,28 +40,29 @@ const argv = yargs
             chalk`{bold Usage}\n  {bold $} {dim <}name{dim >} {dim --}location {dim <}location{dim >} {dim [--folder <folder>]}`
         )
         .positional("name", {
-          describe: `          The name for the new package.`,
+          describe: `The name for the new package.`,
           type: "string",
           demandOption: true
         })
         .option("location", {
           alias: "l",
           describe: `The location for the new package.`,
+          type: "string",
           choices: locations,
           demandOption: true
         })
         .option("folder", {
           alias: "f",
-          describe: chalk`Optional override for the folder name for the package instead of using {dim <}name{dim >}.`
+          describe: chalk`Optional override for the folder name for the package instead of using {dim <}name{dim >}.`,
+          type: "string",
+          demandOption: false
         });
     }
   )
   .demandCommand()
   .version(false)
   .help(false)
   .updateStrings({
-    "Positionals:": chalk.bold("Options"),
-    "Options:": ` `,
```

Comment:
I'm not sure why you insisted on joining these two while still having positional and non-positional arguments (from the way the script was written, despite they were all being processed as positional anyways). Regardless, if you have two separate types of arguments, they should be separated to help out developers understand what is what. By joining these, you're removing contextual help/usage information for a little extra screen space. I encourage you keep this if you choose this PR option.